src/app/components/plus-one/PlusOne.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component, PropTypes } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import * as actions from './actions';
+
+class PlusOne extends Component {
+    constructor(props) {
+        super(props);
+
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+    handleClick() {
+        const { increment } = this.props;
+        increment();
+    }
+
+    render() {
+        const { counter } = this.props;
```

Comment:
We can go either way. We’ll catch and tweak these minor things within code reviews anyway.